testing/testcafe/tests/navigation/form/form.simpleItem.ts

```diff
@@ -37,7 +38,7 @@ fixture`Form`
 
 ['left', 'right', 'top'].forEach((labelLocation) => {
   ['generic.light', 'material.blue.light'].forEach((theme) => {
-    test('widget alignment (T1086611)', async (t) => {
+    safeSizeTest('widget alignment (T1086611)', async (t) => {
```

Comment:
Why do we add `safeSizeTest` while remove it in Sheduler?
Why theme testing is implemented here in old way?

Reply:
A new way will be introduce in the follow PR - https://github.com/DevExpress/DevExtreme/pull/23305/files#top